modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -1,126 +1,132 @@
 
-[#rdbms_requirements]
+== How it works
 
-[WARNING]
-====
+Bonita Migration Tool contains a set of scripts that execute changes on the data to make it compatible with the newer version of Bonita Runtime.
+This tool is provided as a zip archive and contains an executable script that will determine your current configuration and ask you to which version you want to update.
 
-*RDBMS requirements:*
-The version targeted may not support the version of the database that is being migrated. You may then need to upgrade the version of your database prior to running the migration tool.
+The update is step-wise, by maintenance version, and the script manages the sequence of steps from your current version to the target version.
+After each main or maintenance version update, you have the option to pause or continue to the next step.
 
-* Please check the xref:hardware-and-software-requirements.adoc[database requirements].
-* If you need to upgrade your database:
- ** Please make sure to apply all the xref:database-configuration.adoc#specific_database_configuration[RDBMS customisations required by Bonita] when setting up the new version.
-====
+The migration script migrates the following elements:
 
-*Community/Subscription edition:*
-The tool migrates your platform (_bonita_home_ folder and the database). You cannot xref:upgrade-from-community-to-a-subscription-edition.adoc[change edition] while migrating.
+* Engine server
+* Engine database, including all data on active and archived process instances
+* Organization definition
+* Log files from the previous version are not impacted
+* Runtime data (in Bonita Home before Bonita 7.3, in the database after)
+* Configuration files (in Bonita Home before Bonita 7.3, in the database after), replaced with the default configuration files of the new version
 
-* If you are running a Bonita Subscription Pack edition, you need a valid license for your target version.
-* If you are upgrading to a new maintenance version and not changing the minor version number (for example, you are migrating from x.y.0 to x.y.1), your current license remains valid after migration.
+The following are not migrated automatically:
 
-[NOTE]
-====
+* Runtime custom configuration (in Bonita Home before Bonita 7.3, in the database after). Reapply your customizations manually after the migration script has finished. If the target version of the update is Bonita 7.3 or greater, you can use the xref:bonita-bpm-platform-setup.adoc#update_platform_conf[setup tool]).
+* Deployed process definitions: The processes will continue to run using the definition created in the previous version of Bonita.
+* Process definition sources (`.bos` files): Migrate these by importing them into the new version of Bonita Studio.
+* +++<a id="bdm_redeploy">++++++</a>+++Business Data Model, and the business data database: if the updating path includes version `7.0.0`,`7.2.0` or `7.2.4`, the Business Data Model must be redeployed after the update. To do so:
```

Comment:
business data database: if the updating path => business database: if the updating path

Reply:
fixed